_doc/dns.hook

```diff
@@ -63,9 +70,15 @@ updns() {
   (
     declare -f nsupdate_cmds >/dev/null && nsupdate_cmds
     [ -n "$TKIP_KEY" ] && echo key "$TKIP_KEY_NAME" "$TKIP_KEY"
-    echo $op "_acme-challenge.${CH_HOSTNAME}." 60 IN TXT "\"${CH_TXT_VALUE}\""
+    if [[ $op == "add" ]]; then
+        echo update add "_acme-challenge.${CH_HOSTNAME}." 60 IN TXT "\"${CH_TXT_VALUE}\""
+    elif [[ $op == "del" ]]; then
+        echo update delete "_acme-challenge.${CH_HOSTNAME}." IN TXT
+    else
```

Comment:
I don't think this is a good idea. If you do this, then other `_acme-challenge` TXT records for the same domain will be deleted, which could conflict with other challenges being made concurrently on other servers. Unlikely, but there shouldn't be any issue with specifying the TTL and value:

>Deletes any resource records named domain-name. If type and data is provided, only matching resource
           records will be removed. The internet class is assumed if class is not supplied. The ttl is ignored,
           and is only allowed for compatibility.